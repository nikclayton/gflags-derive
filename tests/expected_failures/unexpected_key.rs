extern crate gflags_derive;
use gflags_derive::GFlags;

#[derive(GFlags)]
#[gflags(foo)]
#[allow(dead_code)]
struct Config {
    /// True if log messages should also be sent to STDERR
    to_stderr: bool,

    /// The directory to write log files to
    dir: String,
}

fn main() {}