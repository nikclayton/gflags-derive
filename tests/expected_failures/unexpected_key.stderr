error: Invalid keyword `foo`
 --> $DIR/unexpected_key.rs:5:10
  |
5 | #[gflags(foo)]
  |          ^^^
